spec/system/task_spec.rb

```diff
@@ -1,24 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
Factoryに`association`の定義を記載すると`project_id`の指定が不要になりますね。
```suggestion
  let(:task) { create(:task) }
```

Reply:
b2ffc15 
98c108e
にて変更しました。